typescript/packages/jumble/src/main.tsx

```diff
@@ -2,28 +2,29 @@ import { StrictMode } from "react";
 import { createRoot } from "react-dom/client";
 import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
 import "@/styles/index.css";
-import Home from "@/views/Home.tsx";
 import PhotoFlowIndex from "@/views/experiments/photoflow/Index.tsx";
 import PhotoSetView from "@/views/experiments/photoflow/PhotoSetView.tsx";
 import NewSpell from "@/views/experiments/photoflow/NewSpell.tsx";
 import Shell from "@/views/Shell.tsx";
-import CharmDetail from "@/views/CharmDetail";
 import { CharmsProvider } from "@/contexts/CharmsContext";
+import "./recipes/index";
```

Comment:
I did this in an attempt to get `iterate` working but I'm not really sure how the pieces hang together.